### DISCUSSION TITLE
Google Maps noapi

### DISCUSSION THREAD

**Discussion by 5Y5P0** [AmauriC/tarteaucitron.js#911] [General]
Bonjour, je découvre le service tarteaucitron.js, soyez indulgent s'il vous plaît.

J'ai paramétré le service Google Maps (no api), tout fonctionne bien, acceptation comme refus.
Cependant, quand je regarde la popup tarteaucitron, la zone "Google Maps" m'indique "Autorisé - Ce service n'a déposé aucun cookie" alors qu'il y a bien des cookie "google" déposé.

Y at'il quelquechose que je comprend mal ?

Merci beaucoup aux contributeurs de ce site vraiment très pratique.

**Comment by AmauriC** [comment]
Hello : )

C'est la phrase qui n'est pas très logique. Tarteaucitron ne peux détecter et donc compter que les cookies qui sont déposés directement sur votre site, pas ceux qui le sont sur google.com

Par contre quand Gmaps est désactivé, il n'y a plus de lien vers google.com et donc ces cookies ne sont plus accessibles/visibles (et ceux potentiellement posé sur votre site sont supprimés).

**Reply by 5Y5P0** [reply]
Ok, je comprend mieux, c'est ma méconnaissance du fonctionnement des cookies qui m'a induit en erreur, je supposait que les cookies que je vois lorsque je clique sur le cadena de mon site étaient forcément ceux du site même si ils sont indiqués google.com.

Merci pour vos explications.
